pyart/aux_io/rainbow_wrl.py

```diff
@@ -384,7 +384,7 @@ def _get_angle(ray_info, angle_step=None, scan_type='ppi'):
     else:
         if angle_step is None:
             raise ValueError('Unknown angle step')
-        angle_start = np.array(ray_info['data']*bin_to_deg, dtype='float64')
+        angle_start = np.array(ray_info[0]['data']*bin_to_deg, dtype='float64')
```

Comment:
@kmuehlbauer 
Thanks for clarifying the problem. I see the issue now. You have a newest version of the Rainbow file. I would love to have this information in my own files! :).
I think the best approach is to have a chain of elif:

if (len(ray_info) == 1):
    treatment of case with angle_start only
elif (len(ray_info)==2):
    treatnment of case with angle_start and angle_stop
else:
   treatment of case with all the additional information

I assume that the ray information is given in an specified order so we can keep using indices to access the information in ray_info anyway.

Does this approach make sense to you?